test/src/common/storage/storage-manager.spec.js

```diff
@@ -92,10 +75,8 @@ describe('StorageManager', function() {
     });
   });
 
-  it('should set muted to true/false depends on changed volume', function() {
-    sandbox.stub(StorageWrapper, '_testForLocalStorage').callsFake(() => {
-      StorageWrapper._isLocalStorageAvailable = true;
-    });
+  it('should set muted to true/false depends on changed volume', function () {
+    StorageWrapper._testForLocalStorage = () => (StorageWrapper._isLocalStorageAvailable = true);
```

Comment:
why did you change the stub?
I think it's better to keep it consistently like [here](https://github.com/kaltura/kaltura-player-js/pull/335/files#diff-3847065bb233ba3c3e34f104575a82fcR180)

Reply:
There's no _testForLocalStorage member in StorageWrapper so you can't stub it using sinon (at least with the newest version)